### PR TITLE
Change service name to Monitor

### PR DESCRIPTION
**Pull request by scottaddie**

### CODE REVIEW

File: _data/releases/latest/js-packages.csv [hunk 1]
```diff
@@ -577,7 +577,7 @@
 "@azure/functions-authentication-events","","1.0.0-beta.2","@azure/functions-authentication-events","","NA","NA","NA","compat","false","","","","","","","","","","",""
 "azure-web-pubsub-express","","0.0.1-security.3","Web PubSub CloudEvents Handlers for Express","Web PubSub","NA","NA","NA","compat","false","","","","","","","","","","","Added for MSDocs ToC generation. Not published by azure-sdk automation."
 "@azure/arm-iothub-profile-2020-09-01-hybrid","2.1.0","","@azure/arm-iothub-profile-2020-09-01-hybrid","IoT","iothub","NA","NA","mgmt","true","","","","active","","true","","","","",""
-"azure-monitoring","","0.10.6","azure-monitoring","Resource Management - Monitor","NA","NA","NA","mgmt","false","","","","deprecated","3/31/2023","","@azure/arm-monitor","NA","","",""
+"azure-monitoring","","0.10.6","azure-monitoring","Monitor","NA","NA","NA","mgmt","false","","","","deprecated","3/31/2023","","@azure/arm-monitor","NA","","",""
```
Comment (ronniegeraghty) [inline _data/releases/latest/js-packages.csv:580]: @scottaddie is `azure-monitoring` a data plane or mgmt plane library? If it's a mgmt plane library, noted by the `Type` column being `mgmt` then the `Resource Management -` prefix will get auto added back in the dashboard. 
If `azure-monitoring` is actually a data plane library the `Type` will need to be changed to `client` and the `Replace` column should not point to a mgmt library.

**Comment by scottaddie** [comment]
> @scottaddie why change the name of a package that is already deprecated?

It's just the service name that changed here. "Resource Management - Monitor" isn't the name of an Azure service.

File: _data/releases/latest/js-packages.csv (hunk 1 above)
Comment (scottaddie) [reply _data/releases/latest/js-packages.csv:580]: @ronniegeraghty It's management plane. Why is a "Resource Management -" prefix added to the Service column?

Comment (scottaddie) [reply _data/releases/latest/js-packages.csv:580]: Fixed

**Review by ronniegeraghty — APPROVED** [review]

**Pull request merged**